src/NzbDrone.Common/Http/CurlHttpClient.cs

```diff
@@ -160,12 +167,26 @@ private WebHeaderCollection ProcessHeaderStream(HttpWebRequest webRequest, Strea
             var setCookie = webHeaderCollection.Get("Set-Cookie");
             if (setCookie != null && setCookie.Length > 0 && webRequest.CookieContainer != null)
             {
-                webRequest.CookieContainer.SetCookies(webRequest.RequestUri, setCookie);
+                webRequest.CookieContainer.SetCookies(webRequest.RequestUri, FixSetCookieHeader(setCookie));
             }
 
             return webHeaderCollection;
         }
 
+        private string FixSetCookieHeader(string setCookie)
+        {
+            // fix up the date if it was malformed
+            string pattern = @"(expires=)([^;]+)";
+            Regex expiryMatch = new Regex(pattern, RegexOptions.IgnoreCase | RegexOptions.Compiled);
```

Comment:
please move the regex to a field like we do with the rest of regexes. eg. https://github.com/Sonarr/Sonarr/blob/develop/src/NzbDrone.Core/IndexerSearch/Definitions/SearchCriteriaBase.cs